third_party/nvfuser/csrc/python_frontend/fusion_interface.cpp

```diff
@@ -36,12 +36,8 @@ void FusionInterface::aliasOutputToInput(Nvf::Val* output, Nvf::Val* input) cons
 
 std::vector<at::Tensor> FusionInterface::execute(
     const at::ArrayRef<c10::IValue>& inputs) const {
-  // aliasOutputToInput always adds Tensors as outputs that we don't want
-  // to return to the user. We need to remove them.
-  auto count_output_aliases = fusionPtr()->getOutputAliasIndices().size();
-  auto result = fusionExecutorCachePtr()->runFusionWithInputs(inputs);
-  result.erase(result.begin(), result.begin() + count_output_aliases);
-  return result;
```

Comment:
Oh, it was me who added this hack 🙈

Reply:
Hahaha, now I know whom to blame.

But really it's me not paying enough attention during code review. :)